src/stepper/stepper.ts

```diff
@@ -2501,7 +2508,7 @@ function jsTreeifyMain(
     },
 
     Literal: (target: es.Literal): es.Literal => {
-      if (typeof target.value === 'object') {
+      if (typeof target.value === 'object' && !(target.value === null)) {
```

Comment:
Likewise, can we use `target.value !== null` here?

Reply:
Done